lib/game_team_factory.rb

```diff
@@ -0,0 +1,29 @@
+require "csv"
+
+class GameTeamFactory
+  @@game_teams = []
```

Comment:
I didn't end up using a class variable on mine, but will look forward to revisiting that lesson and possibly refactoring mine to replicate this.